src/main/java/hudson/plugins/im/bot/HealthCommand.java

```diff
@@ -26,17 +26,22 @@
     @Override
     protected CharSequence getMessageForJob(AbstractProject<?, ?> project) {
         StringBuilder msg = new StringBuilder(32);
+        AbstractBuild<?, ?> lastBuild = project.getLastBuild();
         msg.append(project.getFullDisplayName());
         if (project.isDisabled()) {
             msg.append("(disabled)");
         } else if (project.isBuilding()) {
-            msg.append("(BUILDING: ").append(project.getLastBuild().getDurationString()).append(")");
+            msg.append("(BUILDING");
+            if (lastBuild != null) {
+                msg.append(": ");
```

Comment:
OK, re-reading the commit message, @i386 says he had to fix some FB issues so I guess it's one of them. 
NIT: maybe to PR would have been clearer, one to upgrade and fix those, one only for display-url-plugin changes.